Source/Core/Core/Slippi/SlippiNetplay.cpp

```diff
@@ -181,12 +181,12 @@ unsigned int SlippiNetplayClient::OnData(sf::Packet &packet)
 			int32_t headFrame = remotePadQueue.empty() ? 0 : remotePadQueue.front()->frame;
 			int inputsToCopy = frame - headFrame;
 
-			//// Check that the packet actually contains the data it claims to
-			// if((5 + inputsToCopy * SLIPPI_PAD_DATA_SIZE) > (int)packet.getDataSize())
-			//{
-			//	ERROR_LOG(SLIPPI_ONLINE, "Netplay packet too small to read pad buffer");
-			//	break;
-			//}
+			// Check that the packet actually contains the data it claims to
+			if((5 + inputsToCopy * SLIPPI_PAD_DATA_SIZE) > (int)packet.getDataSize())
```

Comment:
Why 5? Maybe should be a constant?

Reply:
Perhaps? But that constant isn't limited to just this PR. It would be needed in a bunch of other locations too.

It's the 1 byte MessageID plus the 4 byte frame index.